docs/content/installation/install_compatibility_requirements.md

```diff
@@ -8,32 +8,30 @@ For IBM Cloud® Satellite users, see [cloud.ibm.com/docs/satellite](https://clou
 
 **Important:** When using Satellite, complete the following checks, configurations, and the installation process before assigning the hosts to your locations. </br>In addition, **do not** create a Kubernetes cluster. Creating the Kubernetes cluster is done through Satellite.
 
--   The CSI driver requires the following ports to be opened on the worker nodes OS firewall:
-    -   **For all iSCSI users**
+The CSI driver requires the following ports to be opened on the worker nodes OS firewall:
+ -   **For all iSCSI users**
 
         Port 3260
 
-    -   **FlashSystem A9000 and A9000R**
+ -   **FlashSystem A9000 and A9000R**
 
         Port 7778
 
-    -   **IBM Spectrum® Virtualize Family includes IBM® SAN Volume Controller and IBM FlashSystem® family members that are built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)**
+ -   **IBM Spectrum® Virtualize family includes IBM® SAN Volume Controller and IBM FlashSystem® family members that are built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)**
 
         Port 22
 
-    -   **DS8000® Family systems**
+ -   **DS8000® family systems**
 
-        Port 8452
-
--   Be sure that multipathing is installed and running.
+      Port 8452
 
 Complete these steps for each worker node in Kubernetes cluster to prepare your environment for installing the CSI (Container Storage Interface) driver.
```

Comment:
not every step needs to be done "for each worker node" (4, 5, 7)

Reply:
those are marked as optional

Reply:
yes, but if a user does opt in any of them, they are not supposed to be done per worker, even though they are under `Complete these steps for each worker node`

Reply:
note that this is not new for 1.8.0, so it is not a merge blocker

Reply:
are the rest per system? (or something else?)

Reply:
the storage-side ones (5.ii. & 7) are per storage system, the orchestrator-cluster-side ones (4 & 5) are per cluster.
6 is per worker node, but it's not mandatory for all worker nodes in order for the feature to work (it can work for a subset of workers)

Reply:
fixed in https://github.com/IBM/ibm-block-csi-driver/pull/466